src/Serializer/SimpleJsonSerializer.php

```diff
@@ -40,27 +49,13 @@ public function meta(array $meta)
 
     public function paginator(PaginatorInterface $paginator)
     {
-        $currentPage = (int)$paginator->getCurrentPage();
-        $lastPage = (int)$paginator->getLastPage();
-
         $pagination = [
-            'total'        => (int)$paginator->getTotal(),
-            'count'        => (int)$paginator->getCount(),
-            'per_page'     => (int)$paginator->getPerPage(),
-            'current_page' => $currentPage,
-            'total_pages'  => $lastPage,
+            'pageNumber' => (int)$paginator->getCurrentPage(),
+            'perPage'    => (int)$paginator->getPerPage(),
+            'pageCount'  => (int)$paginator->getLastPage(),
```

Comment:
should we just force the interface to return `int`s instead of casting this way?

Reply:
Yes, I think that is good.